### PR TITLE
Attempt #826: Linkback CTA text

### LINKED ISSUE #826: Update CTA text to emphasize link between FTM & Find the Makers
Find the Makers became 15% of our total site traffic in its first 48 hours of launch.

Need to update the CTA text on the left-hand side and make it conditional for the FTM index.html or /Maker

Design + Content here: https://www.figma.com/file/qbibGHgcV1LBcpvd2EoHSM/FindTheMasks?node-id=1272%3A9032
image

### PR DESCRIPTION
**Pull request by linderr**
Trying to resolve #826

### CODE REVIEW

**Comment by linderr** [comment]
@ngottlieb @nburt how come when I edit the en.json file and try to add `<a href=""></a>` links it shows text in red even though I am just copying existing code and also labelled the .handlebar file with [html] ?

File: public/i18n/en.json [hunk 1]
```diff
@@ -9,7 +9,11 @@
     "message-documentation": "qqq"
   },
   "ftm-ppe-cta": "Let's get our PPE into the hands of those who need it!",
+  "makers-ppe-cta": "Make PPE & get it into the hands of those who need it.",
   "ftm-requester-near-me": "Search for a requester near you. Not sure how to give? <a class=\"text-light\" href=\"javascript:;\" onclick=\"window.location='faq'+window.location.search;\">Read our FAQ</a>.",
+  "maker-request-near-me": "Coordinate with a Maker Group near you. Questions? <a class=\"text-light\" href=\"javascript:;\" onclick=\"window.location='faq'+window.location.search;\">Read our Maker FAQ.</a>.",
+  "ftm-CTA-link-back": "Want to find requesting institutions near you? <a class=\"text-light" href=\"www.findthemasks.com\">Check out Find the Masks.</a>.",
```
Comment (ngottlieb) [inline public/i18n/en.json]: you're missing the "escape" character on the quote after text-light on both of these, that's why it's red (but also likely causing lots of other issues).

i.e. it needs to be `class=\"text-light\"`

Comment (linderr) [reply public/i18n/en.json]: doh!

File: views/index.handlebars [hunk 2]
```diff
@@ -13,18 +13,24 @@
         <div class="bg-dark-blue text-light">
 
           {{#if (eq dataset "makers")}}
-            <h4>
+            <h4 class="mt-0 i18n" data-i18n="makers-ppe-cta">
               Make PPE &amp; get it into the hands of those who need it.
             </h4>
-            <p>
-              Coordinate with a Maker Group near you. Questions? Read the <a class="text-light" href="/faq#section-makers">Maker FAQ</a>.
+            <p class="i18n" data-i18n="[html]maker-request-near-me">
```
Comment (ngottlieb) [inline views/index.handlebars:19]: let's be consistent about the prefix for `makers` i18n here. @nburt I'm not *that* familiar with the jQuery i18n library -- should these be `ftm-makers-blahblahblah` or is Linda taking the right approach by replacing `ftm` with `makers` as the prefix here?

Comment (linderr) [reply views/index.handlebars:19]: My thinking/prediction was that Maker-only features and thus code demand will outpace the base ftm code given the complex nature of the Makerspace ecosystem IRL. Just attempting to save key strokes :) but happy to whatever

Comment (nburt) [reply views/index.handlebars:19]: Let's keep as is for consistency, can you also update them to all be lowercase too?

All keys should be "ftm-<key in all lowercase characters".

**Comment by linderr** [comment]
@nburt @ngottlieb I think it's okay now! Sorry for all the commits. If I select "draft" will it stop the Smartling PR's from triggering??

Will stick to ftm-xxx conventions and all-lower case - thank you!

I don't know how to do the fancy link-backs yet so it's country-specific
**AKA**` /ca/makers` <--> `/ca `(find the masks)

also country-nav is broken right now on Makers as per #828 so I can't really test/learn

**Comment by linderr** [comment]
I think this is okay now code-wise but the spacing between the two `<p>`'s bothers me lol.

Is there a smart way to keep the two "ftm-maker-xx" strings but limit the spacing between the two? 
b/c i'm just about to hack it and do `<br/>`

image

**Comment by nburt** [comment]
> I think this is okay now code-wise but the spacing between the two `<p>`'s bothers me lol.
> 
> Is there a smart way to keep the two "ftm-maker-xx" strings but limit the spacing between the two?
> b/c i'm just about to hack it and do `<br/>` and have to sacrifice one "ftm-maker-xx" data tag..
> 
> image

You can remove the bottom margin from the first p tag or ya just use a br

**Comment by linderr** [comment]
Oh yay!! it worked!!!
okay, I think this is ready to be merged?

**Comment by linderr** [comment]
hold on. UX/UI wants Title Case --> lower case

**Pull request closed without merging**